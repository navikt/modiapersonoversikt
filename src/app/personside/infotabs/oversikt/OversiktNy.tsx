import { ExternalLinkIcon } from '@navikt/aksel-icons';
import { BodyLong, ExpansionCard, HGrid, HStack, Heading, Link, VStack } from '@navikt/ds-react';
import Card from 'src/components/Card';

const menuItems = [
    {
        title: 'Oppfølging',
        href: '/new/person/oppfolging',
        beskrivelse: 'Kort beskrivelse av hva oppfølging er'
    },
    {
        title: 'Kommunikasjon',
        href: '/new/person/meldinger',
        beskrivelse: 'Kort beskrivelse av hva kommunikasjon er'
    },
    {
        title: 'Arbeid',
        href: '/new/person/arbeid',
        beskrivelse: 'Kort beskrivelse av hva arbeid er'
    },
    {
        title: 'Utbetaling',
        href: '/new/person/utbetaling',
        beskrivelse: 'Kort beskrivelse av hva utbetaling er'
    },
    {
        title: 'Ytelser',
        href: '/new/person/ytelser',
        beskrivelse: 'Kort beskrivelse av hva ytelser er'
    },
    {
        title: 'Dokumenter',
        href: '/new/person/saker',
        beskrivelse: 'Kort beskrivelse av hva dokumenter er'
    },
    {
        title: 'Varsler',
        href: '/new/person/varsler',
        beskrivelse: 'Kort beskrivelse av hva varsler er'
    }
];

const OversiktDetailCard = ({ title, beskrivelse, href }: { title: string; beskrivelse: string; href: string }) => {
    return (
        <Card padding="4">
            <VStack justify="space-between">
                <HStack wrap={false} gap="4" align="top" justify="space-between">
                    <Heading size="medium">{title}</Heading>
                    <div className="border border-gray-400 rounded-md px-2 pt-1">
                        <Link key={title} className="no-underline" href={`${href}`} aria-label={title} variant="action">
                            <ExternalLinkIcon title={`link_til_${title}`} fontSize="1.5rem" />
                        </Link>
                    </div>
                </HStack>
                <p className="mt-4">{beskrivelse}</p>
            </VStack>
        </Card>
    );
};

function OversiktNy() {
    return (
        <VStack gap="4" paddingBlock="2">
            <Card padding="4">
                <VStack align="center">
                    <Heading size="large">Modia personoversikt</Heading>
                    <BodyLong spacing>Kort beskrivelse av hva modia er</BodyLong>
                </VStack>
            </Card>
            <HGrid gap="4" columns={3}>
                {menuItems.map((item) => {
                    return (
                        <OversiktDetailCard
                            key={item.title}
                            title={item.title}
                            beskrivelse={item.beskrivelse}
                            href={item.href}
                        />
                    );
                })}
            </HGrid>

            <ExpansionCard aria-label="Hvor henter vi informasjon fra?">
                <ExpansionCard.Header>
                    <ExpansionCard.Title>Hvor henter vi informasjon fra?</ExpansionCard.Title>
                    <ExpansionCard.Description>Hvilke data henter vi ut, og fra hvor?</ExpansionCard.Description>
                </ExpansionCard.Header>
                <ExpansionCard.Content>
                    <BodyLong spacing>Hvis du er helt eller delvis</BodyLong>
                </ExpansionCard.Content>
            </ExpansionCard>
        </VStack>
    );
}

export default OversiktNy;
