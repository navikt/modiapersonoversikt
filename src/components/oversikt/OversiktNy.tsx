import { ExternalLinkIcon } from '@navikt/aksel-icons';
import { BodyLong, BodyShort, ExpansionCard, HGrid, HStack, Heading, VStack } from '@navikt/ds-react';
import Card from 'src/components/Card';
import { Link } from 'src/components/Link';

const menuItems = [
    {
        title: 'Oppfølging',
        href: '/new/person/oppfolging',
        beskrivelse: 'Kort beskrivelse av hva oppfølging er'
    },
    {
        title: 'Kommunikasjon',
        href: '/new/person/meldinger',
        beskrivelse: 'Kort beskrivelse av hva kommunikasjon er'
    },
    {
        title: 'Arbeid',
        href: '/new/person/arbeid',
        beskrivelse: 'Kort beskrivelse av hva arbeid er'
    },
    {
        title: 'Utbetaling',
        href: '/new/person/utbetaling',
        beskrivelse: 'Kort beskrivelse av hva utbetaling er'
    },
    {
        title: 'Ytelser',
        href: '/new/person/ytelser',
        beskrivelse: 'Kort beskrivelse av hva ytelser er'
    },
    {
        title: 'Dokumenter',
        href: '/new/person/saker',
        beskrivelse: 'Kort beskrivelse av hva dokumenter er'
    },
    {
        title: 'Varsler',
        href: '/new/person/varsler',
        beskrivelse: 'Kort beskrivelse av hva varsler er'
    }
] as const;

const OversiktDetailCard = ({
    title,
    beskrivelse,
    to
}: { title: string; beskrivelse: string; to: (typeof menuItems)[number]['href'] }) => {
    return (
        <Card padding="4">
            <VStack justify="space-between" gap="4">
                <HStack wrap={false} gap="4" justify="space-between">
                    <Heading size="medium">{title}</Heading>
                    <Card flexGrow={0}>
                        <VStack align="center" className="p-2">
                            <Link key={title} className="no-underline" to={to} aria-label={title} variant="action">
                                <ExternalLinkIcon aria-hidden fontSize="1.5rem" />
                            </Link>
                        </VStack>
                    </Card>
                </HStack>
                <BodyShort>{beskrivelse}</BodyShort>
            </VStack>
        </Card>
    );
};

function OversiktNy() {
    return (
        <VStack gap="4" paddingBlock="2">
            <Card padding="4">
                <VStack align="center">
                    <Heading size="large">Modia personoversikt</Heading>
                    <BodyLong spacing>Kort beskrivelse av hva modia er</BodyLong>
                </VStack>
            </Card>
            <HGrid gap="4" columns={3}>
                {menuItems.map((item) => {
                    return (
                        <OversiktDetailCard
                            key={item.title}
                            title={item.title}
                            beskrivelse={item.beskrivelse}
                            to={item.href}
                        />
                    );
                })}
            </HGrid>

            <ExpansionCard aria-label="Hvor henter vi informasjon fra?">
                <ExpansionCard.Header>
                    <ExpansionCard.Title>Hvor henter vi informasjon fra?</ExpansionCard.Title>
                    <ExpansionCard.Description>Hvilke data henter vi ut, og fra hvor?</ExpansionCard.Description>
                </ExpansionCard.Header>
                <ExpansionCard.Content>
                    <BodyLong spacing>Hvis du er helt eller delvis</BodyLong>
                </ExpansionCard.Content>
            </ExpansionCard>
        </VStack>
    );
}

export default OversiktNy;
