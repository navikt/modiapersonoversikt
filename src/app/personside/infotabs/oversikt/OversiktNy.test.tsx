import { render } from '@testing-library/react';
import OversiktNy from 'src/app/personside/infotabs/oversikt/OversiktNy';
import { setupReactQueryMocks } from 'src/test/testStore';
import TestProvider from '../../../../test/Testprovider';

test('Viser ny oversikt med alt innhold', () => {
    setupReactQueryMocks();
    const container = render(
        <TestProvider>
            <OversiktNy />
        </TestProvider>
    );

    const json = container.asFragment();
    expect(json).toMatchSnapshot();
    container.unmount();
});
