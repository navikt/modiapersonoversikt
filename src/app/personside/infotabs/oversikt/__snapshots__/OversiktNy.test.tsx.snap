// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Viser ny oversikt med alt innhold 1`] = `
<DocumentFragment>
  <div
    class="navds-r-pb navds-stack navds-vstack navds-stack-gap navds-stack-direction"
    style="--__ac-r-pb-xs: var(--a-spacing-2); --__ac-stack-gap-xs: var(--a-spacing-4); --__ac-stack-direction-xs: column;"
  >
    <div
      class="navds-r-p navds-r-flex-grow navds-box navds-box-bg navds-box-border-color navds-box-border-width navds-box-border-radius"
      style="--__ac-r-p-xs: var(--a-spacing-4); --__ac-r-flex-grow-xs: 1; --__axc-box-background: var(--ax-bg-raised); --__axc-box-border-color: var(--ax-border-neutral-subtle); --__axc-box-border-width: 1px; --__axc-box-border-radius-xs: var(--ax-border-radius-large);"
    >
      <div
        class="navds-stack navds-vstack navds-stack-align navds-stack-direction"
        style="--__ac-stack-direction-xs: column; --__ac-stack-align-xs: center;"
      >
        <h1
          class="navds-heading navds-heading--large"
        >
          Modia personoversikt
        </h1>
        <p
          class="navds-body-long navds-body-long--medium navds-typo--spacing"
        >
          Kort beskrivelse av hva modia er
        </p>
      </div>
    </div>
    <div
      class="navds-hgrid navds-hgrid-gap"
      style="--__ac-hgrid-gap-xs: var(--a-spacing-4); --__ac-hgrid-columns-xs: repeat(3, minmax(0, 1fr));"
    >
      <div
        class="navds-r-p navds-r-flex-grow navds-box navds-box-bg navds-box-border-color navds-box-border-width navds-box-border-radius"
        style="--__ac-r-p-xs: var(--a-spacing-4); --__ac-r-flex-grow-xs: 1; --__axc-box-background: var(--ax-bg-raised); --__axc-box-border-color: var(--ax-border-neutral-subtle); --__axc-box-border-width: 1px; --__axc-box-border-radius-xs: var(--ax-border-radius-large);"
      >
        <div
          class="navds-stack navds-vstack navds-stack-justify navds-stack-direction"
          style="--__ac-stack-direction-xs: column; --__ac-stack-justify-xs: space-between;"
        >
          <div
            class="navds-stack navds-hstack navds-stack-gap navds-stack-align navds-stack-justify navds-stack-direction"
            style="--__ac-stack-gap-xs: var(--a-spacing-4); --__ac-stack-direction-xs: row; --__ac-stack-align-xs: top; --__ac-stack-justify-xs: space-between;"
          >
            <h1
              class="navds-heading navds-heading--medium"
            >
              Oppfølging
            </h1>
            <div
              class="border border-gray-400 rounded-md px-2 pt-1"
            >
              <a
                aria-label="Oppfølging"
                class="navds-link no-underline navds-link--action"
                href="/new/person/oppfolging"
              >
                <svg
                  aria-labelledby="title-r0"
                  fill="none"
                  focusable="false"
                  font-size="1.5rem"
                  height="1em"
                  role="img"
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <title
                    id="title-r0"
                  >
                    link_til_Oppfølging
                  </title>
                  <path
                    clip-rule="evenodd"
                    d="M20.532 3.471A.75.75 0 0 1 20.75 4v7.5a.75.75 0 0 1-1.5 0V5.81l-8.72 8.72a.75.75 0 1 1-1.06-1.06l8.72-8.72H12.5a.75.75 0 0 1 0-1.5H20c.206 0 .393.083.529.218l.001.002zM4.75 9A.25.25 0 0 1 5 8.75h7a.75.75 0 0 0 0-1.5H5A1.75 1.75 0 0 0 3.25 9v10c0 .966.784 1.75 1.75 1.75h10A1.75 1.75 0 0 0 16.75 19v-7a.75.75 0 0 0-1.5 0v7a.25.25 0 0 1-.25.25H5a.25.25 0 0 1-.25-.25z"
                    fill="currentColor"
                    fill-rule="evenodd"
                  />
                </svg>
              </a>
            </div>
          </div>
          <p
            class="mt-4"
          >
            Kort beskrivelse av hva oppfølging er
          </p>
        </div>
      </div>
      <div
        class="navds-r-p navds-r-flex-grow navds-box navds-box-bg navds-box-border-color navds-box-border-width navds-box-border-radius"
        style="--__ac-r-p-xs: var(--a-spacing-4); --__ac-r-flex-grow-xs: 1; --__axc-box-background: var(--ax-bg-raised); --__axc-box-border-color: var(--ax-border-neutral-subtle); --__axc-box-border-width: 1px; --__axc-box-border-radius-xs: var(--ax-border-radius-large);"
      >
        <div
          class="navds-stack navds-vstack navds-stack-justify navds-stack-direction"
          style="--__ac-stack-direction-xs: column; --__ac-stack-justify-xs: space-between;"
        >
          <div
            class="navds-stack navds-hstack navds-stack-gap navds-stack-align navds-stack-justify navds-stack-direction"
            style="--__ac-stack-gap-xs: var(--a-spacing-4); --__ac-stack-direction-xs: row; --__ac-stack-align-xs: top; --__ac-stack-justify-xs: space-between;"
          >
            <h1
              class="navds-heading navds-heading--medium"
            >
              Kommunikasjon
            </h1>
            <div
              class="border border-gray-400 rounded-md px-2 pt-1"
            >
              <a
                aria-label="Kommunikasjon"
                class="navds-link no-underline navds-link--action"
                href="/new/person/meldinger"
              >
                <svg
                  aria-labelledby="title-r1"
                  fill="none"
                  focusable="false"
                  font-size="1.5rem"
                  height="1em"
                  role="img"
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <title
                    id="title-r1"
                  >
                    link_til_Kommunikasjon
                  </title>
                  <path
                    clip-rule="evenodd"
                    d="M20.532 3.471A.75.75 0 0 1 20.75 4v7.5a.75.75 0 0 1-1.5 0V5.81l-8.72 8.72a.75.75 0 1 1-1.06-1.06l8.72-8.72H12.5a.75.75 0 0 1 0-1.5H20c.206 0 .393.083.529.218l.001.002zM4.75 9A.25.25 0 0 1 5 8.75h7a.75.75 0 0 0 0-1.5H5A1.75 1.75 0 0 0 3.25 9v10c0 .966.784 1.75 1.75 1.75h10A1.75 1.75 0 0 0 16.75 19v-7a.75.75 0 0 0-1.5 0v7a.25.25 0 0 1-.25.25H5a.25.25 0 0 1-.25-.25z"
                    fill="currentColor"
                    fill-rule="evenodd"
                  />
                </svg>
              </a>
            </div>
          </div>
          <p
            class="mt-4"
          >
            Kort beskrivelse av hva kommunikasjon er
          </p>
        </div>
      </div>
      <div
        class="navds-r-p navds-r-flex-grow navds-box navds-box-bg navds-box-border-color navds-box-border-width navds-box-border-radius"
        style="--__ac-r-p-xs: var(--a-spacing-4); --__ac-r-flex-grow-xs: 1; --__axc-box-background: var(--ax-bg-raised); --__axc-box-border-color: var(--ax-border-neutral-subtle); --__axc-box-border-width: 1px; --__axc-box-border-radius-xs: var(--ax-border-radius-large);"
      >
        <div
          class="navds-stack navds-vstack navds-stack-justify navds-stack-direction"
          style="--__ac-stack-direction-xs: column; --__ac-stack-justify-xs: space-between;"
        >
          <div
            class="navds-stack navds-hstack navds-stack-gap navds-stack-align navds-stack-justify navds-stack-direction"
            style="--__ac-stack-gap-xs: var(--a-spacing-4); --__ac-stack-direction-xs: row; --__ac-stack-align-xs: top; --__ac-stack-justify-xs: space-between;"
          >
            <h1
              class="navds-heading navds-heading--medium"
            >
              Arbeid
            </h1>
            <div
              class="border border-gray-400 rounded-md px-2 pt-1"
            >
              <a
                aria-label="Arbeid"
                class="navds-link no-underline navds-link--action"
                href="/new/person/arbeid"
              >
                <svg
                  aria-labelledby="title-r2"
                  fill="none"
                  focusable="false"
                  font-size="1.5rem"
                  height="1em"
                  role="img"
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <title
                    id="title-r2"
                  >
                    link_til_Arbeid
                  </title>
                  <path
                    clip-rule="evenodd"
                    d="M20.532 3.471A.75.75 0 0 1 20.75 4v7.5a.75.75 0 0 1-1.5 0V5.81l-8.72 8.72a.75.75 0 1 1-1.06-1.06l8.72-8.72H12.5a.75.75 0 0 1 0-1.5H20c.206 0 .393.083.529.218l.001.002zM4.75 9A.25.25 0 0 1 5 8.75h7a.75.75 0 0 0 0-1.5H5A1.75 1.75 0 0 0 3.25 9v10c0 .966.784 1.75 1.75 1.75h10A1.75 1.75 0 0 0 16.75 19v-7a.75.75 0 0 0-1.5 0v7a.25.25 0 0 1-.25.25H5a.25.25 0 0 1-.25-.25z"
                    fill="currentColor"
                    fill-rule="evenodd"
                  />
                </svg>
              </a>
            </div>
          </div>
          <p
            class="mt-4"
          >
            Kort beskrivelse av hva arbeid er
          </p>
        </div>
      </div>
      <div
        class="navds-r-p navds-r-flex-grow navds-box navds-box-bg navds-box-border-color navds-box-border-width navds-box-border-radius"
        style="--__ac-r-p-xs: var(--a-spacing-4); --__ac-r-flex-grow-xs: 1; --__axc-box-background: var(--ax-bg-raised); --__axc-box-border-color: var(--ax-border-neutral-subtle); --__axc-box-border-width: 1px; --__axc-box-border-radius-xs: var(--ax-border-radius-large);"
      >
        <div
          class="navds-stack navds-vstack navds-stack-justify navds-stack-direction"
          style="--__ac-stack-direction-xs: column; --__ac-stack-justify-xs: space-between;"
        >
          <div
            class="navds-stack navds-hstack navds-stack-gap navds-stack-align navds-stack-justify navds-stack-direction"
            style="--__ac-stack-gap-xs: var(--a-spacing-4); --__ac-stack-direction-xs: row; --__ac-stack-align-xs: top; --__ac-stack-justify-xs: space-between;"
          >
            <h1
              class="navds-heading navds-heading--medium"
            >
              Utbetaling
            </h1>
            <div
              class="border border-gray-400 rounded-md px-2 pt-1"
            >
              <a
                aria-label="Utbetaling"
                class="navds-link no-underline navds-link--action"
                href="/new/person/utbetaling"
              >
                <svg
                  aria-labelledby="title-r3"
                  fill="none"
                  focusable="false"
                  font-size="1.5rem"
                  height="1em"
                  role="img"
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <title
                    id="title-r3"
                  >
                    link_til_Utbetaling
                  </title>
                  <path
                    clip-rule="evenodd"
                    d="M20.532 3.471A.75.75 0 0 1 20.75 4v7.5a.75.75 0 0 1-1.5 0V5.81l-8.72 8.72a.75.75 0 1 1-1.06-1.06l8.72-8.72H12.5a.75.75 0 0 1 0-1.5H20c.206 0 .393.083.529.218l.001.002zM4.75 9A.25.25 0 0 1 5 8.75h7a.75.75 0 0 0 0-1.5H5A1.75 1.75 0 0 0 3.25 9v10c0 .966.784 1.75 1.75 1.75h10A1.75 1.75 0 0 0 16.75 19v-7a.75.75 0 0 0-1.5 0v7a.25.25 0 0 1-.25.25H5a.25.25 0 0 1-.25-.25z"
                    fill="currentColor"
                    fill-rule="evenodd"
                  />
                </svg>
              </a>
            </div>
          </div>
          <p
            class="mt-4"
          >
            Kort beskrivelse av hva utbetaling er
          </p>
        </div>
      </div>
      <div
        class="navds-r-p navds-r-flex-grow navds-box navds-box-bg navds-box-border-color navds-box-border-width navds-box-border-radius"
        style="--__ac-r-p-xs: var(--a-spacing-4); --__ac-r-flex-grow-xs: 1; --__axc-box-background: var(--ax-bg-raised); --__axc-box-border-color: var(--ax-border-neutral-subtle); --__axc-box-border-width: 1px; --__axc-box-border-radius-xs: var(--ax-border-radius-large);"
      >
        <div
          class="navds-stack navds-vstack navds-stack-justify navds-stack-direction"
          style="--__ac-stack-direction-xs: column; --__ac-stack-justify-xs: space-between;"
        >
          <div
            class="navds-stack navds-hstack navds-stack-gap navds-stack-align navds-stack-justify navds-stack-direction"
            style="--__ac-stack-gap-xs: var(--a-spacing-4); --__ac-stack-direction-xs: row; --__ac-stack-align-xs: top; --__ac-stack-justify-xs: space-between;"
          >
            <h1
              class="navds-heading navds-heading--medium"
            >
              Ytelser
            </h1>
            <div
              class="border border-gray-400 rounded-md px-2 pt-1"
            >
              <a
                aria-label="Ytelser"
                class="navds-link no-underline navds-link--action"
                href="/new/person/ytelser"
              >
                <svg
                  aria-labelledby="title-r4"
                  fill="none"
                  focusable="false"
                  font-size="1.5rem"
                  height="1em"
                  role="img"
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <title
                    id="title-r4"
                  >
                    link_til_Ytelser
                  </title>
                  <path
                    clip-rule="evenodd"
                    d="M20.532 3.471A.75.75 0 0 1 20.75 4v7.5a.75.75 0 0 1-1.5 0V5.81l-8.72 8.72a.75.75 0 1 1-1.06-1.06l8.72-8.72H12.5a.75.75 0 0 1 0-1.5H20c.206 0 .393.083.529.218l.001.002zM4.75 9A.25.25 0 0 1 5 8.75h7a.75.75 0 0 0 0-1.5H5A1.75 1.75 0 0 0 3.25 9v10c0 .966.784 1.75 1.75 1.75h10A1.75 1.75 0 0 0 16.75 19v-7a.75.75 0 0 0-1.5 0v7a.25.25 0 0 1-.25.25H5a.25.25 0 0 1-.25-.25z"
                    fill="currentColor"
                    fill-rule="evenodd"
                  />
                </svg>
              </a>
            </div>
          </div>
          <p
            class="mt-4"
          >
            Kort beskrivelse av hva ytelser er
          </p>
        </div>
      </div>
      <div
        class="navds-r-p navds-r-flex-grow navds-box navds-box-bg navds-box-border-color navds-box-border-width navds-box-border-radius"
        style="--__ac-r-p-xs: var(--a-spacing-4); --__ac-r-flex-grow-xs: 1; --__axc-box-background: var(--ax-bg-raised); --__axc-box-border-color: var(--ax-border-neutral-subtle); --__axc-box-border-width: 1px; --__axc-box-border-radius-xs: var(--ax-border-radius-large);"
      >
        <div
          class="navds-stack navds-vstack navds-stack-justify navds-stack-direction"
          style="--__ac-stack-direction-xs: column; --__ac-stack-justify-xs: space-between;"
        >
          <div
            class="navds-stack navds-hstack navds-stack-gap navds-stack-align navds-stack-justify navds-stack-direction"
            style="--__ac-stack-gap-xs: var(--a-spacing-4); --__ac-stack-direction-xs: row; --__ac-stack-align-xs: top; --__ac-stack-justify-xs: space-between;"
          >
            <h1
              class="navds-heading navds-heading--medium"
            >
              Dokumenter
            </h1>
            <div
              class="border border-gray-400 rounded-md px-2 pt-1"
            >
              <a
                aria-label="Dokumenter"
                class="navds-link no-underline navds-link--action"
                href="/new/person/saker"
              >
                <svg
                  aria-labelledby="title-r5"
                  fill="none"
                  focusable="false"
                  font-size="1.5rem"
                  height="1em"
                  role="img"
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <title
                    id="title-r5"
                  >
                    link_til_Dokumenter
                  </title>
                  <path
                    clip-rule="evenodd"
                    d="M20.532 3.471A.75.75 0 0 1 20.75 4v7.5a.75.75 0 0 1-1.5 0V5.81l-8.72 8.72a.75.75 0 1 1-1.06-1.06l8.72-8.72H12.5a.75.75 0 0 1 0-1.5H20c.206 0 .393.083.529.218l.001.002zM4.75 9A.25.25 0 0 1 5 8.75h7a.75.75 0 0 0 0-1.5H5A1.75 1.75 0 0 0 3.25 9v10c0 .966.784 1.75 1.75 1.75h10A1.75 1.75 0 0 0 16.75 19v-7a.75.75 0 0 0-1.5 0v7a.25.25 0 0 1-.25.25H5a.25.25 0 0 1-.25-.25z"
                    fill="currentColor"
                    fill-rule="evenodd"
                  />
                </svg>
              </a>
            </div>
          </div>
          <p
            class="mt-4"
          >
            Kort beskrivelse av hva dokumenter er
          </p>
        </div>
      </div>
      <div
        class="navds-r-p navds-r-flex-grow navds-box navds-box-bg navds-box-border-color navds-box-border-width navds-box-border-radius"
        style="--__ac-r-p-xs: var(--a-spacing-4); --__ac-r-flex-grow-xs: 1; --__axc-box-background: var(--ax-bg-raised); --__axc-box-border-color: var(--ax-border-neutral-subtle); --__axc-box-border-width: 1px; --__axc-box-border-radius-xs: var(--ax-border-radius-large);"
      >
        <div
          class="navds-stack navds-vstack navds-stack-justify navds-stack-direction"
          style="--__ac-stack-direction-xs: column; --__ac-stack-justify-xs: space-between;"
        >
          <div
            class="navds-stack navds-hstack navds-stack-gap navds-stack-align navds-stack-justify navds-stack-direction"
            style="--__ac-stack-gap-xs: var(--a-spacing-4); --__ac-stack-direction-xs: row; --__ac-stack-align-xs: top; --__ac-stack-justify-xs: space-between;"
          >
            <h1
              class="navds-heading navds-heading--medium"
            >
              Varsler
            </h1>
            <div
              class="border border-gray-400 rounded-md px-2 pt-1"
            >
              <a
                aria-label="Varsler"
                class="navds-link no-underline navds-link--action"
                href="/new/person/varsler"
              >
                <svg
                  aria-labelledby="title-r6"
                  fill="none"
                  focusable="false"
                  font-size="1.5rem"
                  height="1em"
                  role="img"
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <title
                    id="title-r6"
                  >
                    link_til_Varsler
                  </title>
                  <path
                    clip-rule="evenodd"
                    d="M20.532 3.471A.75.75 0 0 1 20.75 4v7.5a.75.75 0 0 1-1.5 0V5.81l-8.72 8.72a.75.75 0 1 1-1.06-1.06l8.72-8.72H12.5a.75.75 0 0 1 0-1.5H20c.206 0 .393.083.529.218l.001.002zM4.75 9A.25.25 0 0 1 5 8.75h7a.75.75 0 0 0 0-1.5H5A1.75 1.75 0 0 0 3.25 9v10c0 .966.784 1.75 1.75 1.75h10A1.75 1.75 0 0 0 16.75 19v-7a.75.75 0 0 0-1.5 0v7a.25.25 0 0 1-.25.25H5a.25.25 0 0 1-.25-.25z"
                    fill="currentColor"
                    fill-rule="evenodd"
                  />
                </svg>
              </a>
            </div>
          </div>
          <p
            class="mt-4"
          >
            Kort beskrivelse av hva varsler er
          </p>
        </div>
      </div>
    </div>
    <section
      aria-label="Hvor henter vi informasjon fra?"
      class="navds-expansioncard navds-expansioncard--medium"
    >
      <div
        class="navds-expansioncard__header"
        data-open="false"
      >
        <div
          class="navds-expansioncard__header-content"
        >
          <h3
            class="navds-expansioncard__title navds-expansioncard__title--medium navds-heading navds-heading--medium"
          >
            Hvor henter vi informasjon fra?
          </h3>
          <p
            class="navds-link-panel__description navds-body-long navds-body-long--medium"
          >
            Hvilke data henter vi ut, og fra hvor?
          </p>
        </div>
        <button
          aria-expanded="false"
          class="navds-expansioncard__header-button"
          type="button"
        >
          <svg
            aria-labelledby="title-r7"
            class="navds-expansioncard__header-chevron"
            fill="none"
            focusable="false"
            height="1em"
            role="img"
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title
              id="title-r7"
            >
              Vis mer
            </title>
            <path
              clip-rule="evenodd"
              d="M5.97 9.47a.75.75 0 0 1 1.06 0L12 14.44l4.97-4.97a.75.75 0 1 1 1.06 1.06l-5.5 5.5a.75.75 0 0 1-1.06 0l-5.5-5.5a.75.75 0 0 1 0-1.06"
              fill="currentColor"
              fill-rule="evenodd"
            />
          </svg>
        </button>
      </div>
      <div
        aria-hidden="true"
        class="navds-expansioncard__content navds-expansioncard__content--closed navds-body-long navds-body-long--medium"
        data-open="false"
      >
        <div
          class="navds-expansioncard__content-inner"
        >
          <p
            class="navds-body-long navds-body-long--medium navds-typo--spacing"
          >
            Hvis du er helt eller delvis
          </p>
        </div>
      </div>
    </section>
  </div>
</DocumentFragment>
`;
